src/main/java/com/fasterxml/jackson/core/util/DefaultPrettyPrinter.java

```diff
@@ -110,10 +110,20 @@ public interface Indenter
      */
     protected String _objectEntrySeparator;
 
+    /**
+     * @since 2.17
```

Comment:
Could you also add a brief explanation of what these do?

Reply:
Hi @cowtowncoder, I fix it

Reply:
Thanks!